airflow/serialization/serialized_objects.py

```diff
@@ -32,8 +32,15 @@
 from airflow.serialization.enums import DagAttributeTypes as DAT, Encoding
 from airflow.serialization.json_schema import Validator, load_dag_schema
 from airflow.settings import json
+from airflow.utils.module_loading import import_string
 from airflow.www.utils import get_python_source
 
+WHITELIST_OPERATOR_EXTRA_LINKS: List[str] = [
```

Comment:
```suggestion
BUILTIN_OPERATOR_EXTRA_LINKS: List[str] = [
```

Its not a quite a whitelist, so how about this as a name?

Reply:
I like this name too. It will change in a moment. I was inspired by this
https://github.com/apache/airflow/blob/master/airflow/plugins_manager.py#L121

Reply:
Done.